pyface/ui/qt4/tasks/split_editor_area_pane.py

```diff
@@ -939,17 +939,17 @@ def get_pixmap(self):
         painter.drawPrimitive(QtGui.QStyle.PE_FrameTabBarBase, optTabBase)
 
         # region of active tab
-        if QtGui.qt_api=='pyqt5':
-            pixmap1 = tabBar.grab()
-        else: # pyqt4 (grabWidget was removed in pyqt5)
+        if QtCore.QT_VERSION >= 0x050000:
+            pixmap1 = tabBar.grab(tab_rect)
```

Comment:
It looks to me as though the missing `tab_rect` in the original code was unintentional: I can't figure out why we'd want it for Qt4 but not for Qt5.